examples/src/main/java/tachyon/examples/BasicNonByteBufferOperations.java

```diff
@@ -61,17 +78,31 @@ public BasicNonByteBufferOperations(TachyonURI masterLocation, TachyonURI filePa
 
   @Override
   public Boolean call() throws Exception {
-    TachyonFS client = TachyonFS.get(mMasterLocation, new TachyonConf());
-
-    write(client, mFilePath, mWriteType, mDeleteIfExists, mLength);
-    return read(client, mFilePath, mReadType);
+    TachyonConf tachyonConf = ClientContext.getConf();
+    tachyonConf.set(Constants.MASTER_HOSTNAME, mMasterLocation.getHost());
+    tachyonConf.set(Constants.MASTER_PORT, Integer.toString(mMasterLocation.getPort()));
+    if (sReinitializer == null) {
+      ClientContext.accessReinitializer(sReinitializerAccesser);
+    }
+    sReinitializer.reinitializeWithConf(tachyonConf);
+    TachyonFileSystem tFS = TachyonFileSystem.get();
```

Comment:
@calvinjia the example runs as a console program with args to specify master address, I assume it is possible to use remote tachyon master, right ?

API has changed a lot since last commit, I'll try to merge with current master branch.